appveyor.yml

```diff
@@ -26,14 +25,15 @@ install:
   - set PATH=%PATH%;C:\Users\appveyor\.cargo\bin
   - set PATH=%PATH%;C:\Users\appveyor\.multirust\toolchains\nightly-x86_64-pc-windows-msvc\lib\rustlib\%TARGET%\lib
   - if NOT "%TARGET%" == "x86_64-pc-windows-msvc" rustup target add %TARGET%
+  # Set custom target in rls.toml config file
+  - if NOT "%TARGET%" == "x86_64-pc-windows-msvc" echo target = "%TARGET%" > rls.toml
```

Comment:
I don't really understand what is going on, either on Windows generally or specifically on Appveyor. What I'd like to do is not specify a target at all and have windows use host == target, just like for other platforms. Is that what happens in real life (i.e., not Appveyor) or not? Is it possible to do here?